corehq/apps/locations/tests/test_reupholster.py

```diff
@@ -198,3 +151,39 @@ def test_path(self):
             parent = make_loc(name, type=type_, parent=parent)
         boston = parent
         self.assertEqual(boston.path, boston.sql_location.path)
+
+
+class TestNoCouchLocationTypes(TestCase):
```

Comment:
the setter is used when you initialize a `Location`, but I just added a couple which change it for an existing location via the setter.